.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -71,16 +71,18 @@ body:
       description: Which Windows versions did you see the issue on?
       multiple: true
       options:
-        - "Insider Build (xxxxx)"
-        - "May 2021 Update (19043)"
-        - "October 2020 Update (19042)"
-        - "May 2020 Update (19041)"
-        - "November 2019 Update (18363)"
-        - "May 2019 Update (18362)"
-        - "October 2018 Update (17763)"
-        - "April 2018 Update (17134)"
-        - "Fall Creators Update (16299)"
-        - "Creators Update (15063)"
+        - "Windows Insider Build (xxxxx)"
+        - "Windows 11: Build 22000"
```

Comment:
I think it would be helpful to use the code names as well for the Windows 10 builds as they are relatively well known: https://en.wikipedia.org/wiki/Windows_10_version_history#Channels

I propose this format:
Windows 10: Build 19044 --> Windows 10 21H2: Build 19044

Reply:
Hey, sound good, but I have a question. What's better to use - windows Version or Codename?

The screenshots, how it looks like:
![image](https://user-images.githubusercontent.com/50652041/154034068-f90ca302-8e0d-49cd-9252-41029eaba7b0.png)

Reply:
I like version — good call!
Windows 11 (21H2): Build 22000

Reply:
I've updated the template with the last commit